broker/src/main/java/org/apache/rocketmq/broker/failover/EscapeBridge.java

```diff
@@ -127,29 +112,74 @@ public PutMessageResult putMessage(MessageExtBrokerInner messageExt) {
         }
     }
 
+    private SendResult putMessageToRemoteBroker(MessageExtBrokerInner messageExt) {
+        final TopicPublishInfo topicPublishInfo = this.brokerController.getTopicRouteInfoManager().tryToFindTopicPublishInfo(messageExt.getTopic());
+        if (null == topicPublishInfo || !topicPublishInfo.ok()) {
+            LOG.warn("putMessageToRemoteBroker: no route info of topic {} when escaping message, msgId={}",
+                messageExt.getTopic(), messageExt.getMsgId());
+            return null;
+        }
+
+        final long invokeID = RandomUtils.nextLong(0, Long.MAX_VALUE);
```

Comment:
I wonder what invokeID does here? Can it be removed?

Reply:
I'll remove it for it is unnecessary.